src/main/java/frc/robot/subsystems/elevator/Elevator.java

```diff
@@ -199,7 +211,9 @@ public boolean elevatorAboveInterferenceZone() {
 
   @Override
   public void periodic() {
-    controlPosition(desiredPosition);
+    if (setRealDesired) {
+      controlPosition(desiredPosition);
+    }
   }
```

Comment:
If the elevator is above the intake, then robot gets disabled, then elevator drifts down onto the intake. I'm a little worried about enabling and clicking home, which would deploy the intake. Intake would be in contact with the elevator while it moves which could be bad. We should take a look at that.